dags/services/AWSService.py

```diff
@@ -0,0 +1,40 @@
+import boto3
```

Comment:
Nit: I'm generally in favor of module names being snake_case. -> `aws_service.py` or `nextflow_tower_service.py`.  

Since we have plans on moving this to sagetasks and using airflow provided operators, this isn't a big deal.

Reply:
changed in latest commit